issuesync.rb

```diff
@@ -80,11 +80,14 @@ def format(io, issue, comments)
 
     def format_body(io, issue)
       title = "##{issue.number}: #{issue.title.strip}"
+      title << issue.labels
```

Comment:
At the moment I'm doing `head -n1 -q issues/*.md | grep -v CLOSED` to get a list of all the issues, then putting it in a text editor. By having the labels on the same line I still get the issues in my text editor and can search for them - so it has some mild convenience for me. That said, I could just as easily `head -n2` and then remove the newline.